universe/Pathfinder.cpp

```diff
@@ -1010,6 +1015,51 @@ std::multimap<double, int> Pathfinder::PathfinderImpl::ImmediateNeighbors(int sy
 }
 
 
+void Pathfinder::PathfinderImpl::WithinJumpsCacheHit(
+    std::unordered_set<int>* result, size_t jump_limit,
+    size_t ii, const distance_matrix_storage<short>::row_ref row) const {
```

Comment:
That is the row index, typically written  `i`.  However `ii` is much easier to grep.  This one I'd like to keep.